.prettierrc.js

```diff
@@ -4,4 +4,13 @@ module.exports = {
   singleQuote: true,
   trailingComma: 'es5',
   printWidth: 100,
+
+  overrides: [
+    {
+      files: '*.hbs',
+      options: {
+        parser: 'glimmer',
+      },
+    },
+  ],
```

Comment:
why do we want to run prettier on any templates in this repo?

Reply:
Running it without specifying the parser will cause an unpleasant non-breaking error when `tests/dummy/app/templates/application.hbs` is encountered.

Would you prefer we whitelist `.hbs` files?